### ISSUE TITLE
How to run AnimateDiff on base Stable-diffusion?

### ISSUE THREAD

**Issue by aniki-ly** [guoyww/AnimateDiff#105]
I've noticed that the motion modules are trained on the stable-diffusion model and subsequently applied to other customized models. However, when I tested the script, specifically 'animated.py', using the base stable-diffusion model, the results were unsatisfactory. Could you provide some guidance on how to resolve this issue?

**Comment by haofanwang** [comment]
Indeed, it is quite weird. I conjecture that `mm_sd_v15.ckpt` is problematic (but not sure why). I have tested stable-diffusion-1.5 as base with `mm_sd_v14.ckpt` and `mm_sd_v15.ckpt` where all params are set to the same, you can find that the latter is much worse.

`mm_sd_v14.ckpt`
![sample](https://github.com/guoyww/AnimateDiff/assets/18741068/0b98231b-f5be-498f-913a-56f36da76611)

`mm_sd_v15.ckpt`
![sample (1)](https://github.com/guoyww/AnimateDiff/assets/18741068/652ac308-21f5-479a-a74a-beb12d275267)

**Comment by patrolli** [comment]
I tried using stable diffusion 1.4 as the base model and then performed inference using motion module v1.4. It appears that the single-frame quality of sd-1.4 is better, but it's not as smooth as 1.5 (not sure if it's due to a different sampling rate during training).
